### PR TITLE
TCP Native Transport properties for Vert.x HttpServerOptions

### PR DESCRIPTION
**Pull request by KolomiietsSerhii**
Details about SO_REUSEPORT: https://man7.org/linux/man-pages/man7/socket.7.html
Deatils about TCP_QUICKACK: https://man7.org/linux/man-pages/man7/tcp.7.html

### CODE REVIEW

File: src/main/java/org/prebid/server/spring/config/WebConfiguration.java [hunk 1]
```diff
@@ -83,16 +83,19 @@ HttpServerOptions httpServerOptions(
...
 
         final HttpServerOptions httpServerOptions = new HttpServerOptions()
                 .setHandle100ContinueAutomatically(true)
                 .setMaxInitialLineLength(maxInitialLineLength)
                 .setMaxHeaderSize(maxHeaderSize)
                 .setCompressionSupported(true)
                 .setDecompressionSupported(true)
-                .setIdleTimeout(10); // kick off long processing requests
-
+                .setIdleTimeout(10) // kick off long processing requests
```
Comment (SerhiiNahornyi) [inline src/main/java/org/prebid/server/spring/config/WebConfiguration.java:96]: Looks like this one should be configurable

Comment (KolomiietsSerhii) [reply src/main/java/org/prebid/server/spring/config/WebConfiguration.java:96]: Sure, not a problem.

Comment (KolomiietsSerhii) [reply src/main/java/org/prebid/server/spring/config/WebConfiguration.java]: Done.

**Review by Net-burst — COMMENTED** [review]
I have a proposal for those parameters: make them optional. Meaning that I propose to fall back to what is configured system-wide. You'll need to change how they are defined though due to as `null` becomes a valid option because of this.
@SerhiiNahornyi what do you think?

**Comment by SerhiiNahornyi** [comment]
With PRs like this, our main mission is not to brake production for already running PBS users.
So, agree with @Net-burst we need to update this behaviour

**Review by SerhiiNahornyi — APPROVED** [review]

**Pull request merged**